libtelemetry/src/main/java/com/mapbox/android/telemetry/CertificateBlacklist.java

```diff
@@ -112,19 +127,23 @@ private long retrieveLastUpdateTime() {
     return lastUpdateTime;
   }
 
-  void updateBlacklist() {
+  void updateBlacklist(HttpUrl requestUrl) {
```

Comment:
this method name is misleading, should we pick a better name for it? also this method does two things: 1) builds request and 2) enqueues request. could we make it a little more readable?